src/app/applicationComponents/Main/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import NewsList from '../../reusableComponents/NewsList';
+import newsAppConfig from '../../configFiles/newsAppConfig';
+
+class Main extends React.Component {
+    render() {
+        return (
+            <Switch>
+                <Route
+                    exact
+                    path="/"
+                    render={() => (
+                        <main
+                            className={`${
+                                this.props.className
+                            } container-fluid`}
+                        >
+                            <section className="row">
```

Comment:
I think this news list is a separate component too

Reply:
Done.
Created new component - NewsColumns.